sqllineage/core/parser/sqlfluff/extractors/create_insert.py

```diff
@@ -27,42 +27,50 @@ class CreateInsertExtractor(BaseExtractor):
     ]
 
     def extract(
-        self,
-        statement: BaseSegment,
-        context: AnalyzerContext,
+            self,
+            statement: BaseSegment,
+            context: AnalyzerContext,
     ) -> SubQueryLineageHolder:
         holder = self._init_holder(context)
-        src_flag = tgt_flag = False
+        src_flag = tgt_flag = col_flag = False
         for segment in list_child_segments(statement):
```

Comment:
child_segments = list_child_segments(statement)
for idx, segment in enumerate(child_segments):
    ...